Source/HTTP/httpcallresponse.cpp

```diff
@@ -23,10 +22,26 @@ HCHttpCallResponseSetResponseString(
     _In_ PCSTR_T responseString
     )
 {
-    verify_http_singleton();
     call->responseString = responseString;
-    HC_TRACE_INFORMATION(HTTPCLIENT, "HCHttpCallResponseSetResponseString [ID %llu]: responseString=%s",
-        call->id, responseString);
+
+#if HC_TRACE_ENABLE
+
+#if HC_CHAR_T_IS_WIDE
+    size_t len = wcslen(responseString);
+#else
+    size_t len = strlen(responseString);
+#endif
+    if (len > 2048)
```

Comment:
consider using %.2048s to limit the number of characters printed

Reply:
Ah, that worked